source/Module/STIG/Functions.Checklist.ps1

```diff
@@ -592,34 +686,44 @@ function Get-FindingDetailsString
     }
     return $returnString
 }
+<#
+    .SYNOPSIS
+        Extracts the node targeted by the MOF file
+
+#>
 function Get-TargetNodeFromMof
 {
-    [OutputType([string])]
+    [OutputType([String])]
     [CmdletBinding()]
     param
     (
-        [Parameter(Mandatory)]
-        [string]
-        $MofString
+        [Parameter(Mandatory = $true)]
+        [String]
+        $mofString
```

Comment:
Parameter names should begin with a capital letter.

Reply:
Done.